deployment/ccip/view/v1_5/token_pool.go

```diff
@@ -0,0 +1,81 @@
+package v1_5
+
+import (
+	"encoding/hex"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+
+	"github.com/smartcontractkit/chainlink/deployment/common/view/types"
+)
+
+type TokenPoolContract interface {
```

Comment:
This is technically the interface for v1_5_1, package name should be updated as there are breaking changes between v1_5 and v1_5_1

Reply:
Also, add comment for interface & maybe link here: https://github.com/smartcontractkit/chainlink/blob/develop/core/gethwrappers/ccip/generated/token_pool/token_pool.go

So it is clear where these methods are coming from.